### PR TITLE
Freeze coloredlogs to save tests

### LINKED ISSUE #2252: Fix cli download tests
Because the test fails all the time when Ensembl is down

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug. fix #2252

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by github actions

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ## [x.x.x]
 ### Added
 ### Fixed
+- Freeze coloredlogs temporarily
```
Comment (northwestwitch) [inline CHANGELOG.md:10]: Famous last words..

**Review by northwestwitch — APPROVED** [review]
👍

**Comment by dnil** [comment]
Fix #2252..

File: CHANGELOG.md (hunk 1 above)
Comment (dnil) [reply CHANGELOG.md:10]: 🤣 I know. I felt it as soon as I hit commit.

**Pull request merged**